src/lightning_app/utilities/scheduler.py

```diff
@@ -24,6 +26,7 @@ def run(self) -> None:
                 self._exit_event.wait(self._sleep_time)
                 self.run_once()
         except Exception as e:
+            logger.error(traceback.print_exc())
```

Comment:
Forcing traceback to be printed to logs is a bit intrusive, but I guess this is our only chance to do it on this thread?